exp/applications.md

```diff
@@ -42,11 +42,11 @@ In general, the bootstrap process goes through the following steps in order:
 
 ![Application bootstrap process|571x653](https://assets.ubuntu.com/v1/7eed04fd-application-bootstrap.png)
 
-If one of the steps fails, AMS will interrupt the bootstrap process and make the entire process fail. As a result, the status of the base container will be marked with `error` and the application's status will end up with `error` as well.
+If one of the steps fails, AMS will interrupt the bootstrap process and make the entire process fail. As a result, the status of the base container will be set to `error` and the application status is set to `error` as well.
 
-[note type="information" status="Note"]An application crash or ANR upon APK installation will cause the bootstrap process to terminate abnormally and the status of application is set to `error` too.[/note]
+Remember that an application crash or ANR upon APK installation will cause the bootstrap process to terminate abnormally and the application status is set to `error`.
```

Comment:
```suggestion
Remember that an application crash or ANR upon APK installation will cause the bootstrap process to terminate abnormally and the application status will be set to `error`.
```

As you're using the future tense in the beginning of the sentence, I think you should also use it in the other parts.

Reply:
Actually present tense is better, switched everything to present tense.